packages/manager/src/features/IAM/Shared/AssignedRolesTable/ChangeRoleDrawer.tsx

```diff
@@ -157,25 +143,34 @@ export const ChangeRoleDrawer = ({ onClose, open, role }: Props) => {
           <Link to=""> Learn more about roles and permissions.</Link>
         </Typography>
 
-        <Typography sx={{ marginBottom: 2.5 }}>
-          Change from role{' '}
-          <span style={{ font: theme.font.bold }}>{role?.name}</span> to:
+        <Typography sx={{ marginBottom: theme.tokens.spacing.S12 }}>
+          Change from role <strong>{role?.name}</strong> to:
         </Typography>
 
         <Controller
-          render={({ field }) => (
+          render={({ field, fieldState }) => (
             <Autocomplete
+              renderInput={(params) => (
+                <TextField
+                  {...params}
+                  errorText={fieldState.error?.message}
+                  hiddenLabel
+                  label=""
+                  noMarginTop
+                />
+              )}
               label="Assign New Roles"
               loading={accountPermissionsLoading}
               onChange={(_, value) => field.onChange(value)}
               options={allRoles}
               placeholder="Select a Role"
               textFieldProps={{ hideLabel: true, noMarginTop: true }}
-              value={field.value}
+              value={field.value || null}
             />
           )}
           control={control}
           name="roleName"
+          rules={{ required: 'Role is required' }}
```

Comment:
```suggestion
          rules={{ required: 'Role is required.' }}
```
Nit: adding a period is a nice-to-have

Reply:
sure, done